geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/sortedset/AbstractZAddIntegrationTest.java

```diff
@@ -152,18 +152,72 @@ public void zaddCountsOnlyNewMembers_givenMultipleCopiesOfTheSameMember() {
 
   @Test
   public void zaddCountsOnlyNewMembers_givenMultipleCopiesOfTheSameMember_toAnExistingSet() {
-    Long addCount = (Long) jedis.zadd(SORTED_SET_KEY, 1.0, "otherMember");
+    Long addCount = jedis.zadd(SORTED_SET_KEY, 1.0, "otherMember");
     assertThat(addCount).isEqualTo(1);
     jedis.sendCommand(SORTED_SET_KEY, Protocol.Command.ZADD, SORTED_SET_KEY,
         "1", "member", "2", "member", "3", "member");
     assertThat(jedis.zcard(SORTED_SET_KEY)).isEqualTo(2);
     assertThat(jedis.zscore(SORTED_SET_KEY, "member")).isEqualTo(3.0);
   }
 
+  @Test
+  public void zaddDoesNotCountExistingMembersWithoutChanges_whenCHSpecified() {
+    Map<String, Double> initMap = makeMemberScoreMap(2 * INITIAL_MEMBER_COUNT, 0);
+    Long addCount = jedis.zadd(SORTED_SET_KEY, initMap);
+    assertThat(addCount).isEqualTo(2 * INITIAL_MEMBER_COUNT);
+    assertThat(jedis.zcard(SORTED_SET_KEY)).isEqualTo(2 * INITIAL_MEMBER_COUNT);
```

Comment:
These assertions aren't really necessary here, since that behaviour is tested elsewhere, so they can be removed to make the test simpler. The assertion on `zcard` at the end of the test is still good to have though, to confirm that the CH option isn't making `zadd` behave differently somehow.